.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js:
+  - "node"
+before_script:
+  npm install -g express remark-lint remark-cli remark-preset-lint-consistent remark-preset-lint-markdown-style-guide remark-preset-lint-recommended
+script: remark --rc-path $TRAVIS_BUILD_DIR/.remarkrc $TRAVIS_BUILD_DIR/docs/source
```

Comment:
Should we run both the link checker and the linter as part of the travis configuration?

Reply:
Sure. It might make sense to run the sphinx build also.